source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,127 @@
-Your First Task Script (NOT YET WRITTEN)
+Your First Task Script
 ----------------------
 
-**TODO: Write-up that guides the user through writing a basic task script (just a task with a narrate line) using VS-Code, emphasizing things like file extension and placement. Also run it in-game with `/ex run`.**
+<!-- **TODO: Write-up that guides the user through writing a basic task script (just a task with a narrate line) using VS-Code, emphasizing things like file extension and placement. Also run it in-game with `/ex run`.** -->
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+## Task script basics
```

Comment:
This and all other titles: titlecase!

Reply:
Also all headers MUST be three or more `#` symbols.

Reply:
Titlecase is inconsistent with modern documentation standards.
![2019-09-05 21_52_30-Release Notes_ 1 38 0 - Denizen-Beginners-Guide - Visual Studio Code](https://user-images.githubusercontent.com/774463/64397620-837fbf00-d027-11e9-874c-fcd1e13417af.png)

Reply:
Please update the contribution guidelines. They currently state:

```
Sub-headings in files must always be at least 3 hash symbols (that is, ###). One or two will be considered a main heading and find their want into the table-of-contents where they don't belong.
```

This is ambiguous; I read it as three ### being for sub-headings *within* an article, and that one or two would put the header in the table of contents *for that article.*

The distinction is clear when explained; it should be further clarified on the guidelines.